README.md

```diff
@@ -11,88 +11,3 @@ See [what's deployed](https://whatsdeployed.io/s-J18)
 
 ## Code of Conduct
 By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/). If you need to report a problem, please see our [CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md) guide.
-
-## Development
```

Comment:
My suggestion would be to leave this in the README for ease of use. Alternatively we should have a link to the relevant part of the documentation.